### PR TITLE
Nr: Rewrite/Edit/Touchup 05_02, 06_a, 07_a

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
So this is the next installment of the Rewrite/Edit/Touch up(I don't care what we call it.)

I decided to try to do more than one file in each PR. It was easier for me. I hope it doesn't make it harder for reviewers. If it is, sorry because I already finished editing the entire scenario, and it's waiting in branches.

Same plan as before, a lot of the changes should be self-explanatory, however for the ones that I thought weren't I have a changelog with the file and the line.

One more thing to add, is the narrator from the first few scenarios is back. No offense to whoever originally wrote him, but the narrator is really hard to understand. I rewrote quite a bit of it. If you don't think I understood what was going on perfectly/ think of a better way to write it, you're welcome to suggest changes.

Changelog:

05_a_02:

Rewrote the wording on line 163

I deleted an entire dialog paragraph because it just repeated things

06_a

I removed the () in 23 and changed clash because it sounds like fighting, not arming.

The "are they" in 302 didn't sound very natural.

I noticed in the style guide that in 410 500 should be five-hundred.

Rewrote the dialog in 507 to make it sound like it was said by someone who had been fighting for three days straight.

07:
I changed the wording in 436 because a "huh" didn't seem like the best statement.

In line 488 I didn't think "Let's go" was needed.

### CODE REVIEW

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 1]
```diff
@@ -16,10 +16,10 @@
...
         [/part]
         [part]
-            story= _ "As they approached they were met with a most welcome sight: new fortifications, looking very much like the legendary impenetrable dwarvish defenses of old, had sprung up everywhere. All of them were manned by alert and well-armed dwarves who hailed them boisterously. The mines were belching smoke from the furiously working forges. New weapons and armor for humans and dwarves alike lay neatly stacked, and the deafening clang of hammers and picks from the dwarves promised more."
+            story= _ "As they approached they were met with a most welcome sight: new fortifications, looking very much like the legendary, impenetrable dwarvish defenses of old, had sprung up everywhere. All of them were manned by alert and well-armed dwarves who hailed them boisterously. Smoke belched from furiously working forges and the heat from the smelters was nearly staggering. New weapons and armor for humans and dwarves alike lay neatly stacked, and the deafening clang of hammers and picks from the dwarves promised more."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:22]: The first comma is unnecessary I think but the rest is great.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 2]
```diff
@@ -16,10 +16,10 @@
     [story]
         [part]
             music=loyalists.ogg
-            story= _ "Laden with gold and filled with an elation at their victory that was tempered with sadness at the death of their friends, Tallin and his people made their way back to the mines."
+            story= _ "Laden with gold, and filled with not only elation at their victory, but sadness for their departed comrades, Tallin and his people made their way back to the mines."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:19]:
```suggestion
            story= _ "Laden with gold, and filled with both elation at their victory and sadness for their departed comrades, Tallin and his people made their way back to the mines."
```

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 3]
```diff
@@ -24,37 +24,37 @@
...
+        {STORY_PART_HAMEL (_ "Aye. But we must utterly crush this orcish host if we are to ever ha' peace.")}
 
         {STORY_PART_HAMEL (_ "Orcs are fickle; if we negotiated a treaty with them — assuming they would accept — it would just be broken as soon as the next warlord arose among them.")}
 
         {STORY_PART_TALLIN (_ "Then what shall we do? Can we assassinate this Rakshas fellow and throw them into disarray once more?")}
 
         #po: "coil" meant, in Middle and Early Modern English, a difficulty
         #po: or problem.
-        {STORY_PART_HAMEL (_ "That will be very difficult. We did that last time and they will no doubt be alert to that threat now. Anyway, it would be but a temporary solution. Soon another strong leader would emerge and we’d find oursel’ back in the same coil.")}
+        {STORY_PART_HAMEL (_ "That will be very difficult. We did that last time, and they will no doubt be alert to that threat now. Anyway, it would be but a temporary solution. Soon another strong leader would come along, and we’d find oursel’ back in the same coil.")}
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:49]: The first comma isn't needed.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 4]
```diff
@@ -24,37 +24,37 @@
...
 
         {STORY_PART_TALLIN (_ "Well, back to the drawing board. What should we do?")}
 
         {STORY_PART_HAMEL (_ "I think we should try to gather more allies. That way we can engage them on many fronts, or divide them into smaller groups, or cut them off from their supplies. The possibilities are endless.")}
 
         {STORY_PART_TALLIN (_ "Wise words, but who is there in these wild Northlands that might aid us?")}
 
-        {STORY_PART_HAMEL (_ "You can count the elves out. They are a selfish lot, only interested in themselves. They would only aid us if something drastic happened to them and they needed our help. What about Wesnoth?")}
+        {STORY_PART_HAMEL (_ "You can count the elves out. They are a selfish lot, only interested in themselves. They would only aid us if something drastic happened to them, and they needed our help. What about Wesnoth?")}
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:57]: Ditto, doesn't need the comma I think.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 5]
```diff
@@ -369,7 +369,7 @@
 
         [message]
             speaker="Ro'Sothian"
-            message= _ "Leave <i>you</i> in peace?! It is your kind that won’t leave <i>us</i> in peace! Ever since you blasted dwarves built that damn road you have harassed us unceasingly! Now, it ends here! After I kill you I will split your carcass and hang it at the mouth of this pass as a warning for any other fools like you! And unlike us <i>you</i> won’t come back."
+            message= _ "Leave <i>you</i> in peace?! It is your kind that won’t leave <i>us</i> in peace! Ever since you blasted dwarves built that damn road you have harassed us unceasingly! Now, it ends here! After I kill you I will split your carcass and hang it at the mouth of this pass as a warning for any other fools like you! And, unlike us, <i>you</i> won’t come back."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:372]:
```suggestion
            message= _ "Leave <i>you</i> in peace?! It is your kind that won’t leave <i>us</i> in peace! Ever since you blasted dwarves built that damn road you have harassed us unceasingly! Now, it ends here! After I kill you I will split your carcass and hang it at the mouth of this pass as a warning for any other fools like you! And unlike us, <i>you</i> won’t come back."
```

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 6]
```diff
@@ -433,7 +433,7 @@
 
         [message]
             speaker="Ro'Sothian"
-            message= _ "Huh?! He killed me!"
+            message= _ "Why not? He killed me!"
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:436]: I like the original more.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 7]
```diff
@@ -835,7 +835,7 @@
 
         [message]
             speaker="Ro'Arthian"
-            message= _ "Very well, Tallin. We accept under the terms that your people will forever leave us in peace after this."
+            message= _ "Very well, Tallin. We accept under the terms that after this your people will forever leave us in peace ."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:838]: I don't think there's much of a difference so the original is fine.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg (hunk 1 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:22]: Is it legendary and impenetrable, or is it t_the_ legendary "impenetrable dwarvish defenses" TM?

Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:22]: Since this isn't a formal document with stricter grammar rules, I air on the side of readability and speaking patterns especially if it's dialogue. So just because it's grammatically correct doesn't mean it's a change we should make.

Just FYI for further text revisions.

Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:22]: I understand that. This isn't my first time editing stories.

However, A comma shouldn't affect readability.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 3 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:49]: Ok.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 4 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:57]: Yeah, the comma is wrong here.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 6 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:436]: I like the new ones, if other people agree with you however, I'll change it.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg (hunk 1 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:22]: For sure, but I do think commas impact readability since people naturally tend to pause when encountering them. I don't think it's a huge deal either way but depends on the author's style. So if it's between the author and an editor I will prefer to go with the author's version.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 7 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:838]: You can wait for other feedback first.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg (hunk 1 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg]: Either way, it isn't a big step-down in editing. I changed it.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 7 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:838]: Sure go for it.

**Comment by TheShadowOfHassen** [comment]
A bit of help for someone who tried to read the failed check logs and couldn't figure it out.

It says:
`Game error: Quoted string not terminated`
` at campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:226`

 But I don't see the issue.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 8]
```diff
@@ -160,7 +160,7 @@
 
         [message]
             speaker=Hamel
-            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some of the dwarves who are expert with the sword, mace and bow, as well as some of yer own men who have grown skilled by experience, can teach others how to wield them."
+            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some dwarves are experts in the sword, mace, and bow and can aid your veterans in teaching others.
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:163]: You left off the end quotation here, check your other ones too.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 9]
```diff
@@ -174,14 +174,9 @@
             message= _ "Aye, it’s true we do prefer our axes and hammers, but then again, we make swords. We couldna’ claim to be expert weaponsmiths wi’out kenning how to wield the weapons we make."
         [/message]
 
-        [message]
```
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:177]: I suppose one could argue this is redundant, but I find it adds to the 'flavour' of what Hamel is saying. I think it would be a shame to get rid of this entirely without anything to compensate.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 7 above)
Comment (Wedge009) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:838]: Either works for me, but if you go ahead with this change please remove the space from before the full-stop.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg (hunk 8 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:163]: That's the only one. Thanks! (I'm assuming it's like python and runs errors not at the same place?)

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 7 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg]: I made it a full stop.

**Comment by TheShadowOfHassen** [comment]
I need help with the error message. I looked at the line. Nothing.

**Comment by Wedge009** [comment]
> Changes not staged for commit

No idea for this one. Seems the CI system somehow has got into a state where it's got uncommitted changes.

I was going to suggest earlier that instead of pushing commits one at a time that you just complete a block of work and push your commits in one go. Pushing one at a time also causes dozens of notifications, including e-mails.

**Comment by TheShadowOfHassen** [comment]
I'm _new-ish_ to GitHub.  So no clue. I just use GitHub Desktop.

I tried a stupid fix to see if it fixes it, if not I can delete the two commits I made.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg (hunk 9 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:177]: I don't understand what you mean.

Comment (Wedge009) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:177]: I'd prefer to leave this in.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 10]
```diff
@@ -55,7 +55,7 @@
         {STORY_PART_TALLIN (_ "Wise words, but who is there in these wild Northlands that might aid us?")}
 
         {STORY_PART_HAMEL (_ "You can count the elves out. They are a selfish lot, only interested in themselves. They would only aid us if something drastic happened to them and they needed our help. What about Wesnoth?")}
-
+
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:58]: ... and the other trailing whitespace is here.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 11]
```diff
@@ -16,10 +16,10 @@
     [story]
         [part]
             music=loyalists.ogg
-            story= _ "Laden with gold and filled with an elation at their victory that was tempered with sadness at the death of their friends, Tallin and his people made their way back to the mines."
+            story= _ "Laden with gold, and filled with both elation at their victory and sadness for their departed comrades, Tallin and his people made their way back to the mines."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:19]: Should probably have a `#po:` comment for "departed comrades". It's clear to a native speaker, but it's also an idiom that some might not have met.

Might also want to comment about the importance of the gold to the later plot, thus why it's appearing before the mention of the dead.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 12]
```diff
@@ -145,7 +145,7 @@
 
         [message]
             speaker=Galim
-            message= _ "We have cleared the connecting tunnels to here and to the old keep as well as other parts o’ Knalga. It is slow and dangerous work, mostly due to the trolls. But it ha’ been made easier by Malifor’s ending."
+            message= _ "We have cleared the connecting tunnels from here to the old keep as well as other parts o’ Knalga. It is slow and dangerous work, mostly due to the trolls. But it ha’ been made easier by Malifor’s destruction."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:148]: I like the change to "from".

"Destruction" is awkwardly ambiguous, it can be read both as "it's been made easier now Malifor is out of the way" and "it's been made easier because Malifor killed a lot of trolls too".

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 13]
```diff
@@ -160,7 +160,7 @@
 
         [message]
             speaker=Hamel
-            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some of the dwarves who are expert with the sword, mace and bow, as well as some of yer own men who have grown skilled by experience, can teach others how to wield them."
+            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some dwarves are experts in the sword, mace, and bow and can aid your veterans in teaching others."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:163]: I don't like either of them, but would prefer the old one of the two.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 14]
```diff
@@ -173,15 +173,15 @@
...
+        
         [message]
             speaker=Hamel
             message= _ "There are some dwarves out there that are bonny fighters wi’ a sword as any ye are like to meet. Belike with bows, maces and any other weapons we fashion. In fact, Galim here is quite skilled in the art."
         [/message]
 
         [message]
             speaker=Tallin
-            message= _ "That’s great! What is the price of your weapons? We have gathered much booty from Malifor’s treasury and we will be able to pay you generously."
+            message= _ "That’s great! What is the price of your weapons? We have gathered much booty from Malifor’s treasury and will be able to pay you generously."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:184]: "booty" doesn't feel right to me. I think this line either should be rewritten completely, or not change at all.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 15]
```diff
@@ -191,7 +191,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "But don’t you need the gold to help rebuild Knalga? And we have plenty."
+            message= _ "But don’t you need the gold to help rebuild Knalga? We have plenty."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:194]: Feels like an unnecessary change to me. My preference would be to remove everything after the question mark, however that would need rewrites elsewhere in the dialog, because the undocumented purpose of this (IIUC) is to draw the player's attention to the amount of gold that they have.

File: data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg [hunk 16]
```diff
@@ -235,7 +235,7 @@
         [message]
             id=Rakshas
             # wmllint: local spelling Pah
-            message= _ "Pah! I would like to see those cowardly dwarves try their treachery on me as they did on the Black-Tusk! In any case, those dwarves will soon be groveling at my feet and equipping my people with the finest weapons. With the dwarvish weapons, and the wealth of Knalga, nothing will be able to stop us. Even that little backwater that you humans call Wesnoth will be nothing but a burnt out, pillaged wreck before long."
+            message= _ "Pah! I would like to see those cowardly dwarves try their treachery on me as they did on the Black-Tusk! In any case, those dwarves will soon be groveling at my feet and equipping my people with the finest weapons. With the dwarvish weapons, and the wealth of Knalga, nothing will be able to stop us. Even that little backwater that you humans call Wesnoth will be nothing but a burnt out, pillaged wreck before long!"
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg:238]: Both seem similar to me, it works with a full-stop because Rakshas is just stating facts as he sees them. Thus my preference is for no change, rather than leaving the translators to find a single-character change in a long string.

File: data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg [hunk 17]
```diff
@@ -299,7 +299,7 @@
 
         [message]
             id=Rakshas
-            message= _ "So! Your forces are managing to advance upon me, are they? Very impressive, but it shall do you no good. KNIGHTS!!"
+            message= _ "So! Your forces are managing to advance upon me? Very impressive, but it shall do you no good. KNIGHTS!!"
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg:302]: Not a comment on the change, but a comment on this event - I think the entire riders-on-hills ambush should be rewritten, either to give the player hints about the tomato surprise, or to give them a fair retreat.

File: data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg [hunk 18]
```diff
@@ -504,7 +504,7 @@
         [message]
             id=Tallin
             # wmllint: local spelling Geez
-            message= _ "Geez, we have been fighting them for three whole days and nights now with no end in sight. Now the fourth night is approaching, what should we do?"
+            message= _ "Geez, we have been fighting them for three whole days, and there's still as many orcs as ever. What should we do?"
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg:507]: Both seem the same to me.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 19]
```diff
@@ -24,29 +24,29 @@
             story= _ "The dwarves were more than ready for them, however, and as soon as the humans got through the defenses the well-prepared dwarves fell on the battle-weary and bloodied orcs."
         [/part]
         [part]
-            story= _ "Wave after wave of orcs crashed onto the dwarven defenses and shattered."
+            story= _ "Wave after wave of orcs crashed onto the dwarven defenses only to be shattered."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:27]: Both seem the same to me.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 20]
```diff
@@ -406,7 +406,7 @@
 
                 [message]
                     speaker="Ro'Sothian"
-                    message= _ "Camerin! Great Lords of Darkness, I haven’t seen you in ages! Here, let me just finish off this little vermin and I’ll be right over."
+                    message= _ "Camerin! Great Lords of Darkness, I haven’t seen you in ages! Here, let me just finish off this little vermin, and I’ll be right over."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:409]: I prefer the original, the lack of pause matches the time that he's expecting the action to take.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 21]
```diff
@@ -752,12 +752,12 @@
 
         [message]
             speaker="Ro'Arthian"
-            message= _ "What do <i>I</i> want? I’ll tell you what I want! I want the stinking carcasses of you, all your henchmen and those dwarvish vermin off my land! And <i>stay</i> off!"
+            message= _ "What do <i>I</i> want? I’ll tell you what I want! I want the stinking carcasses of you, all your henchmen and those dwarvish vermin off my land!"
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:755]: I prefer the original for an old man's rant.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 22]
```diff
@@ -835,7 +835,7 @@
 
         [message]
             speaker="Ro'Arthian"
-            message= _ "Very well, Tallin. We accept under the terms that your people will forever leave us in peace after this."
+            message= _ "Very well, Tallin. We accept under the terms that after this your people will forever leave us in peace."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:838]: I prefer the original.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg (hunk 14 above)
Comment (Wedge009) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg]: I think I mentioned that before in another issue/PR. I think @nemaara dismissed it on the basis that NR has a more laid-back tone overall. Too, it's out-of-scope for this PR. For the change specifically, I think it's okay either way.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg (hunk 13 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:163]: Can you make an attempt to switch this text to dwarvish speak?

File: data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg (hunk 16 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg:238]: Okay we can leave this one as the original for the sake of translators please.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 3 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:49]: Please remove the first comma you forgot lol.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 20 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:409]: Okay please revert to original.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 6 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:436]: Please revert to original, the "Huh?" feels more indignant to me.

**Comment by TheShadowOfHassen** [comment]
Some help with the check error please?

File: data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg (hunk 17 above)
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg:302]: Could add a po comment that "knights" means "wolf riders" here.

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 23]
```diff
@@ -753,11 +753,12 @@
         [message]
             speaker="Ro'Arthian"
             message= _ "What do <i>I</i> want? I’ll tell you what I want! I want the stinking carcasses of you, all your henchmen and those dwarvish vermin off my land! And <i>stay</i> off!"
+
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:756]: The failing check is because of this blank line

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 24]
```diff
@@ -160,7 +160,7 @@
 
         [message]
             speaker=Hamel
-            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some of the dwarves who are expert with the sword, mace and bow, as well as some of yer own men who have grown skilled by experience, can teach others how to wield them."
+            message= _ "A' you kin see, Tallin, there are muckle heaps o' them, a' ready to be borne inna battle. An, some o' us dwarves are experts in the sword, mace, and bow and can aid yer veterans in teaching others."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:163]:
```suggestion
            message= _ "As ye ken see, Tallin, there be muckle heaps o' them, all ready to be borne inta battle. And, some o’ us dwarves be experts in the sword, mace, and bow and ken aid yer veterans in teaching others."
```

Also please add a po comment again in regular english for translators :)

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg (hunk 22 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:838]: Please leave the original text.

File: data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg [hunk 25]
```diff
@@ -299,6 +299,7 @@
 
         [message]
             id=Rakshas
+            # po: knight's means wolf riders
```
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg:302]: Plurals should not include an apostrophe.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 26]
```diff
@@ -160,7 +160,7 @@
 
         [message]
             speaker=Hamel
-            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some of the dwarves who are expert with the sword, mace and bow, as well as some of yer own men who have grown skilled by experience, can teach others how to wield them."
+            message= _ "As ye ken see, Tallin, there be muckle heaps o' them, all ready to be borne inta battle. And, some o’ us dwarves be experts in the sword, mace, and bow and ken aid yer veterans in teaching others."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:163]: Still needs the po comment in regular english, then I can merge.

File: data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg [hunk 27]
```diff
@@ -160,7 +160,8 @@
 
         [message]
             speaker=Hamel
-            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some of the dwarves who are expert with the sword, mace and bow, as well as some of yer own men who have grown skilled by experience, can teach others how to wield them."
+            # po: As you can see, Tallin there are a lot of them ready to be born into battle. Some of us dwarves are experts in the sword, mace, and bow and can aid your veterans in teaching others.
+            message= _ "As ye ken see, Tallin, there be muckle heaps o' them, all ready to be borne inta battle. And, some o’ us dwarves be experts in the sword, mace, and bow and ken aid yer veterans in teaching others."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg:164]: Just realized this apostrophe needs to be the unicode one.

```suggestion
            message= _ "As ye ken see, Tallin, there be muckle heaps o’ them, all ready to be borne inta battle. And, some o’ us dwarves be experts in the sword, mace, and bow and ken aid yer veterans in teaching others."
```

File: data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg [hunk 28]
```diff
@@ -504,7 +505,7 @@
         [message]
             id=Tallin
             # wmllint: local spelling Geez
-            message= _ "Geez, we have been fighting them for three whole days and nights now with no end in sight. Now the fourth night is approaching, what should we do?"
+            message= _ "Geez, we have been fighting them for three whole days, and there's still as many orcs as ever. What should we do?"
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg:508]:
```suggestion
            message= _ "Geez, we have been fighting them for three whole days, and there’s still as many orcs as ever. What should we do?"
```

File: data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg [hunk 29]
```diff
@@ -24,29 +24,29 @@
...
 
-        {STORY_PART_TALLIN (_ "It was fierce and bloody. We fought them for three days and nights with no end in sight. Every time we killed an orc, it seemed that two more would take his place. We most assuredly cannot win fighting like this.")}
+        {STORY_PART_TALLIN (_ "It was fierce and bloody. We fought them for three days and nights. Every time we killed an orc, it seemed that two more would take his place. We most assuredly cannot win fighting like this.")}
 
         {STORY_PART_HAMEL (_ "So you withdrew. Rightly done, me lad. Our prepared defenses will be able to hold the orcs off far longer and with less loss than ye could ha’ done in the open field. I see that my training of you hasna’ been for naught.")}
 
         {STORY_PART_TALLIN (_ "And I am grateful — but what should we do from here? The number of humans and dwarves is limited, unlike the orcs whose numbers seem unlimited.")}
 
-        {STORY_PART_HAMEL (_ "Aye. But we must utterly crush this orcish host if we are to ha’ anything other than a brief respite.")}
+        {STORY_PART_HAMEL (_ "Aye. But we must utterly crush this orcish host if we are to ever ha' peace.")}
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg:41]:
```suggestion
        {STORY_PART_HAMEL (_ "Aye. But we must utterly crush this orcish host if we are to ever ha’ peace.")}
```

**Review by nemaara — APPROVED** [review]
Waiting till CI passes just in case, then merging.

**Comment by TheShadowOfHassen** [comment]
Yay!
